MapboxNavigationTests/InstructionsBannerViewTests.swift

```diff
@@ -117,7 +145,10 @@ extension InstructionsBannerViewTests {
     
     func verifyView(_ view: UIView, size: CGSize) {
         view.frame.size = size
-        FBSnapshotVerifyView(view)
+        // Allow the UI to download images
+        DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + 2) {
```

Comment:
We should mock these HTTP requests to avoid depending on the server to respond in time.